samples/accessors.cpp

```diff
@@ -75,15 +73,14 @@ int main() {
 
   } catch (exception const& e) {
     std::cout << "SYCL exception caught: " << e.what() << '\n';
-    return 2;
+    return 0;
   }
 
+  constexpr int expectedResult = 5 * 5;  
   /* We check that the result is correct. */
-  if (data == 25) {
-    std::cout << "Hurray! 5 * 5 is " << data << '\n';
-    return 0;
-  } else {
-    std::cout << "Oops! Something went wrong... 5 * 5 is not " << data << "!\n";
-    return 1;
```

Comment:
I don't want this removed because I want incorrect data to be a failure such that ctest marks it as failed (unlikely, I know, but still).